File: player.class.rb
class Player

	def initialize(nameput, health, attack, defense, speed, job)
		@name = nameput
		@hp = health
		@atk = attack
		@def = defense
		@spd = speed
		@job = job
	end

	def getName
		return @name
	end

	def getJob
		return @job
	end

	def getAttack
		return @atk
	end

	def getHealth
		return @hp
	end

	def getStamina
		return @sp
	end

	def getDefense
		return @def
	end

	def getSpeed
		return @spd
	end

	def attack
		puts @name + " attacks the enemy!"
		return @atk
	end

	def takeDamage(attack)
		@hp -= attack - @def
		damage = attack - @def
		puts @name + " took " + damage.to_s + " amount of damage. Current health: " + @hp.to_s
		return attack - @def
	end
end

File: player_class.py
class Player():

	def __init__(self, nameput, health, attack, defense, speed, job):
		self.name = nameput
		self.hp = health
		self.atk = attack
		self.df = defense
		self.spd = speed
		self.job = job

	def getName(self):
		return self.name

	def getJob(self):
		return self.job

	def getAttack(self):
		return self.atk

	def getHealth(self):
		return self.hp

	def getStamina(self):
		return self.sp

	def getDefense(self):
		return self.df

	def getSpeed(self):
		return self.spd

	def attack(self):
		print(self.name + " attacks the enemy!")
		return self.atk

	def takeDamage(self, attack):
		self.hp -= attack - self.df
		damage = attack - self.df
		print(self.name + " took " + str(damage) + " amount of damage. Current health: " + str(self.hp))
		return attack - self.df
